test/indexer/test_tdr.py

```diff
@@ -181,6 +181,23 @@ def test_fetch_bundle(self):
         with self.assertRaises(RequirementError):
             self._test_fetch_bundle(bundle, load_tables=False)
 
+    def test_disallow_double_pipe_joiner(self):
+        bundle = self._canned_bundle(self.source, self.bundle_uuid)
+        self._test_fetch_bundle(bundle, load_tables=True)
+
+        dataset = self.source.spec.bq_name
+        project_table = self.tinyquery.tables_by_name[dataset + '.project']
+        project_content_column = project_table.columns['content'].values
+        project_content = json.loads(one(project_content_column))
+        project = first(project_content['contributors'])
+
+        assert project['institution'] == 'Lund University'
+        project['institution'] += f' {config.manifest_column_joiner} LabMED'
+        project_content_column[0] = json.dumps(project_content)
+
+        with self.assertRaisesRegex(RequirementError, r'`||` is disallowed'):
```

Comment:
Abraham, as I said before, `|` carries special meaning in regexes. What you wrote is a regex that matches a single backtick, the empty string or `` ` is disallowed ``